modules/social_features/social_lazy_loading/social_lazy_loading.info.yml

```diff
@@ -0,0 +1,7 @@
+name: 'Social Lazy Loading'
```

Comment:
This is technically not a feature. Perhaps move to modules/custom instead?

Reply:
Good point! Done!